pom.xml

```diff
@@ -766,5 +766,56 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <!-- Github Packages Integration -->
+            <id>github-packages</id>
+            <distributionManagement>
+                <!-- Deploy to Github Packages -->
+                <repository>
+                    <id>github-packages</id>
+                    <name>Github iSantePlus Packages</name>
+                    <url>https://maven.pkg.github.com/isanteplus/isanteplus-packages</url>
+                    <uniqueVersion>false</uniqueVersion>
+                </repository>
+                <snapshotRepository>
+                    <id>github-packages</id>
+                    <name>Github iSantePlus Packages</name>
+                    <url>https://maven.pkg.github.com/isanteplus/isanteplus-packages</url>
+                    <uniqueVersion>true</uniqueVersion>
+                </snapshotRepository>
+            </distributionManagement>
+            <repositories>
+                <!-- Use the Github Packages Repo first when looking up dependencies -->
+                <repository>
+                    <id>github-packages</id>
+                    <name>Github iSantePlus Packages</name>
+                    <url>https://maven.pkg.github.com/isanteplus/isanteplus-packages</url>
+                </repository>
+            </repositories>
+            <build>
+                <plugins>
+                    <!-- Disable possible test jar generation -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-jar-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-pmd-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>validate</id>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
Ditto. PS I think I separately disabled `test-jar`s in [this commit](https://github.com/openmrs/openmrs-module-fhir2/commit/bd57419d45feadab98554b338eb4b3b8c0fdeba2).